source/help/apps/desktop-guide.rst

```diff
@@ -0,0 +1,176 @@
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Toggle window visibility when clicking on the tray icon                       | v1.3.0 +                  | N/A                       | N/A                       | `magicmonty <https://github.com/magicmonty>`_                                                                                                          |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Leave app running in notification center when application window is closed    | N/A                       | v1.3.0 +                  | v1.3.0 +                  | `magicmonty <https://github.com/magicmonty>`_ (OS X); `yuya-oc <https://github.com/yuya-oc>`_ (Linux)                                                  |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Show red badge on taskbar icon to indicate unread messages                    | v1.4.0 +                  | v1.4.0 +                  | N/A                       | `Razzeee <https://github.com/Razzeee>`_                                                                                                                |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Flash taskbar icon when a new message is received                             | v1.3.0 +                  | N/A                       | v1.4.0 +                  | `MetalCar <https://github.com/metalcar>`_ (Windows); `jnugh <https://github.com/jnugh>`_ (Linux)                                                       |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+
+**Hide menu bar**
```

Comment:
Propose using headings rather than bold, so anchor links are automatically generated. Visually they can look similar, e.g.: 

#### This is Heading 4 format 

**This is bold**